src/pages/WantedPlus/WantedPlusContents.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import PlusCard from './components/PlusCard';
+
+function WantedPlusContents() {
+  const [contentsData, setContentsData] = useState([]);
+  useEffect(() => {
+    fetch('/data/plusContents.json')
+      .then(res => res.json())
+      .then(res => setContentsData(res));
+  }, []);
```

Comment:
```suggestion
  const [contents, setContents] = useState([]);

  useEffect(() => {
    fetch('/data/plusContents.json')
      .then(res => res.json())
      .then(res => setContentsData(res));
  }, []);
```

data등의 접미사는 사용안하는게 오히려 더 좋습니다! 어차피 모든 건 다 데이터니까요

그리고 공백 통해서 가독성 향상시켜주면 좀 더 좋겠네요!